.travis.yml

```diff
@@ -8,11 +8,11 @@ matrix:
   allow_failures:
     - go: master
 env:
-  - NODE_VERSION="8.9.4" DEP_VERSION="0.4.1"
+  - NODE_VERSION="8.9.4"
```

Comment:
I was thinking actually to move the SDK to using a fixed version so we dont get our CI broken by random changes in dep. 

Any reason why we would use the build from latest master ? as they in their readme, we should use that only for hacking around.

Reply:
Oh did they document that?  I missed it.  If this is their recommendation then yes let's do that.

Reply:
they say in their Readme that if you want to hack around it ( presumably to develop on it) otherwise they point to their releases.  They don't do a release on every master change.